ietf/meeting/views.py

```diff
@@ -279,15 +279,22 @@ def materials_document(request, document, num=None, ext=None):
         if len(file_ext) == 2 and file_ext[1] == '.md' and mtype == 'text/plain':
             sorted_accept = sort_accept_tuple(request.META.get('HTTP_ACCEPT'))
             for atype in sorted_accept:
-                if atype[0] == 'text/markdown':
-                    content_type = content_type.replace('plain', 'markdown', 1)
-                    break;
-                elif atype[0] == 'text/html':
-                    bytes = "<html>\n<head><base target=\"_blank\" /></head>\n<body>\n%s\n</body>\n</html>\n" % markdown.markdown(bytes.decode(encoding=chset))
-                    content_type = content_type.replace('plain', 'html', 1)
-                    break;
-                elif atype[0] == 'text/plain':
-                    break;
+                if atype[0] == "text/markdown":
+                    content_type = content_type.replace("plain", "markdown", 1)
+                    break
+                elif atype[0] == "text/html":
+                    bytes = render(
```

Comment:
This is building an HttpResponse object, not a string (thanks to @kesara for spotting it).